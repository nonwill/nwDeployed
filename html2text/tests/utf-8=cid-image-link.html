<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
<head>
<title> Bestelling bevestigd </title>
</head>
<body style="margin: 0; padding: 0; -webkit-text-size-adjust: 100%; -ms-text-size-adjust: 100%; background-color: #eeeeee;" bgcolor="#eeeeee">
                                          <img height="auto" src="cid:3dff0d6344f912415ae595da15ec08e9@kokenenwonen" style="line-height: 100%; -ms-interpolation-mode: bicubic; border: 0; display: block; outline: none; text-decoration: none; height: auto; width: 100%; font-size: 13px;" width="100%" border="0"></a>
</body>
</html>

