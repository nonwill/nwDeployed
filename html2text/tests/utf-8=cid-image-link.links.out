[1]


############ RReeffeerreenncceess ############
   1. cid:3dff0d6344f912415ae595da15ec08e9@kokenenwonen
